Ecierge.Uno.Controls/GroupedComboBox.cs

```diff
@@ -118,90 +270,285 @@ public GroupedComboBox()
     /// <inheritdoc/>
     protected override void OnApplyTemplate()
     {
+        this.SelectionChanged -= GropedComboBox_SelectionChanged;
+        this.LostFocus -= (s, e) =>
+        {
+            if (textBox is null || dropDownButton is null || popupIfEditable is null || popupIfNotEditable is null)
+                return;
+            if (textBox.FocusState == FocusState.Unfocused && dropDownButton.FocusState == FocusState.Unfocused && IsEditable && popupIfEditable.FocusState == FocusState.Unfocused)
+            {
+                popupIfEditable.IsOpen = false;
+                popupIfNotEditable.IsOpen = false;
+                isKeyDown = false;
+            }
+        };
+        if (textBox is not null)
+        {
+            textBox.RemoveHandler(KeyDownEvent, new KeyEventHandler(ItemsHost_KeyDown));
```

Comment:
Делегат можно и не писать
```suggestion
            textBox.RemoveHandler(KeyDownEvent, ItemsHost_KeyDown);
```